frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePullBranchFastForwardOnlyAction.java

```diff
@@ -99,6 +101,10 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       }
 
       if (FetchUpToDateTimeoutStatus.isUpToDate(gitRepository)) {
+        VcsNotifier.getInstance(project).notifyInfo(null,
```

Comment:
Pls name the param
```suggestion
        VcsNotifier.getInstance(project).notifyInfo(/* displayId */ null,
```

(at least IIRC that's `displayId`)

Reply:
Note that this probably won't pass the binary compat check against 2020.1 ☹️

Reply:
...which is soon to be un-supported on #837

Reply:
so wait for ... #837 ? because as I see all notifyInfo is available from 2020.3

Reply:
Or even better, base your PR off @mkondratek 's branch

Reply:
Which also implies waiting until #837 is merged to develop, but you get more relevant feedback from the CI

Reply:
Hmm now the PR is based on #837 but the commits are still based on `develop`... pls actually rebase the commits on the top of `git-machete-tab-on-git-init`!

Reply:
Generally, a branch layout like that:

![image](https://user-images.githubusercontent.com/3383210/168382257-a60adbbf-d701-4f19-883b-a440e462bddd.png)